project/deps.sc

```diff
@@ -50,11 +50,12 @@ object InternalDeps {
 object Deps {
   object Versions {
     // jni-utils version may need to be sync-ed when bumping the coursier version
-    def coursier      = "2.1.0-M5-18-gfebf9838c"
-    def jsoniterScala = "2.13.8"
-    def scalaMeta     = "4.5.1"
-    def scalaNative   = "0.4.4"
-    def scalaPackager = "0.1.26"
+    def coursier              = "2.1.0-M5-18-gfebf9838c"
+    def jsoniterScala         = "2.13.8"
+    def scalaJsEnvJsdomNodejs = "1.1.0"
```

Comment:
NIT Do we really need to put this version here? It's only used once, so it can just be inlined where it's used.